EKF/control.cpp

```diff
@@ -419,45 +418,34 @@ void Ekf::controlOpticalFlowFusion()
 			}
 		}
 
-		// Handle cases where we are using optical flow but are no longer able to because data is old
-		// or its use has been inhibited.
-		if (_control_status.flags.opt_flow) {
-		       if (_inhibit_flow_use) {
-			       _control_status.flags.opt_flow = false;
-			       _time_last_of_fuse = 0;
-
-			} else if ((_time_last_imu - _time_last_of_fuse) > (uint64_t)_params.reset_timeout_max) {
-				_control_status.flags.opt_flow = false;
-
-			}
+		// Handle cases where we are using optical flow but are no longer able to because
+		// its use has been inhibited.
+		if (_control_status.flags.opt_flow && _inhibit_flow_use) {
+			ECL_WARN("Optical flow data quality poor - stopping use");
```

Comment:
_inhibit_flow_use can toggle on and off as the vehicle goes between hover and moving if it is also using GPS. Because of this this message may flood the interface.

Reply:
Also this ability to switch flow use on and off depending on vehicle movement (when it is also using GPS) is a feature and does not necessarily indicate a fault or bad flow data, just an operating condition where it is better to not use flow.